build-apisix-openresty-centos7.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -euo pipefail
+set -x
+
+sed -i '/--prefix="$OR_PREFIX"/a\    --with-cc-opt="-I$openssl_prefix/include" \\\n    --with-ld-opt="-L$openssl_prefix/lib -Wl,-rpath,$openssl_prefix/lib" \\' build-apisix-openresty.sh
```

Comment:
Please pass it as an argument instead of modifying the script

Reply:
that means modify the `build-apisix-openresty.sh` to adding places to get parameters?